Logger/IEntity.cs

```diff
@@ -1,6 +1,7 @@
 ﻿namespace Logger;
+
 public interface IEntity
 {
-
-
+    Guid ID { get; init; }
+    string Name { get; init; }
```

Comment:
```suggestion
    string Name { get; }
```
This doesn't need an init setter on it, just a getter is fine!